polkadot.js

```diff
@@ -0,0 +1,727 @@
+      },
+      {
+        "id": 14,
+        "imageUrl": "https://i.imgur.com/ZtKYg13.png",
+        "title": "Their future together",
+        "details": "Together, Polkadot and Kusama form a dynamic duo that encourages fearless exploration and experimentation. Picture your parachains as connections between the island blockchains, constantly growing and evolving, ready to connect with other blockchains and adapt to the ever-changing tides. This allows your parachains to continuously grow and change over time, evolving into something better. The only limit is how fast you can evolve!"
+      },
+      {
+        "id": 15,
+        "choices": [
+          "Kusama is a Cosmos app-chain",
+          "Kusama is a canary network for Polkadot",
+          "Polkadot is a fork of Kusama/Dotsama"
+        ],
+        "correctChoice": "Kusama is a canary network for Polkadot",
+        "title": "What is the relationship between Polkadot and Kusama/Dotsama?"
+      },
+      {
+        "id": 16,
+        "imageUrl": "https://i.imgur.com/pnFudYZ.png",
+        "title": "Polkadot pioneer!",
+        "details": "Look at you! You’ve taken your first steps with Polkadot and are now able to understand why it’s used, who uses it, and how to use it. It’s really much simpler than you think! We’ve established that Polkadot is super versatile, and so are its tools! Let’s now have a look at how to work with Polkadot!"
+      },
+      {
+        "id": 17,
+        "tweetReason": "Share the love on Twitter - we'll be rewarding the best ones with swag and prizes! ❤️",
+        "tweetContent": "I'm building on @Polkadot with @easya_app on my #60DaysOfPolkadot!\n\nCurrently learning about... \n\n🧑‍💻 How parachains work!\n⛓ How to build blockchains with @substrate_io \n🖼 How to speed up development 10x",
+        "tweetContentUrl": ""
+      }
+    ]
+  },
+  {
+    "title": "How to work with Polkadot",
+    "index": 2,
+    "challengeSteps": [
+      {
+        "id": 1,
+        "imageUrl": "https://i.imgur.com/Ys0BGmG.jpg",
+        "title": "Gear up with Polkadot",
+        "details": "To start building your own parachain on the Polkadot network, you need a Polkadot account. This account lets you manage your DOT tokens, participate in staking and governance, and access the tools you need to build your parachain."
+      },
+      {
+        "id": 2,
+        "imageUrl": "https://i.imgur.com/zzQnxmD.png",
+        "title": "Safeguard your assets",
+        "details": "As you gather resources on the island, wallets become essential tools for managing and protecting your DOT tokens and other assets within the Polkadot ecosystem. Choose from various types of wallets, such as hardware, software and browser-based wallets, each with its own benefits and security features. Safeguard your digital treasures in popular wallets like Enkrypt, PolkaGate, SubWallet and Talisman."
```

Comment:
The flagship wallets in Polkadot are:

- Mobile: Nova Wallet
- Browser: Talisman
- Hardware: Ledger